### PR TITLE
Replace relative CSS units with with fixed values [KHCP-5598]

### PR DESCRIPTION
**Pull request by mptap**
# Summary
JIRA: https://konghq.atlassian.net/browse/KHCP-5598

* Replace all rem units with px
* Replace all em units with px
* Remove relative units error downgrade in stylelint [KHCP-5605]]

## PR Checklist

* [x] Does not introduce dependencies
* [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [x] **Tests pass:** check the output of yarn test
* [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [x] **Framework style:** abides by the essential rules in Vue's style guide
* [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

[KHCP-5605]: https://konghq.atlassian.net/browse/KHCP-5605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/.vitepress/components/KSkeletonBox.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<template>
```
Comment (adamdehaven) [inline docs/.vitepress/components/KSkeletonBox.vue:1]: @mptap why did you need to recreate this component in the docs? That shouldn't be needed

File: src/components/KDropdownMenu/KDropdownMenu.vue [hunk 2]
```diff
@@ -222,9 +222,9 @@ export default defineComponent({
     margin-left: var(--spacing-xs, spacing(xs));
     vertical-align: middle;
     content: "";
-    border-top: 0.325em solid;
-    border-right: 0.325em solid transparent;
-    border-left: 0.325em solid transparent;
+    border-top: 5.2px solid;
+    border-right: 5.2px solid transparent;
+    border-left: 5.2px solid transparent;
```
Comment (adamdehaven) [inline src/components/KDropdownMenu/KDropdownMenu.vue:227]: suggestion: if we just round these up to `6px` does it still look ok? 

Same suggestion for all occurrences (e.g. KSelect)

File: src/components/KViewSwitcher/KViewSwitcher.vue [hunk 3]
```diff
@@ -80,8 +80,8 @@ export default defineComponent({
 
   .icon {
     position: relative;
-    width: 1.5rem;
-    height: 1.5rem;
+    width: 18px;
+    height: 18px;
```
Comment (adamdehaven) [inline src/components/KViewSwitcher/KViewSwitcher.vue:84]: Should these be `24px` instead?

Comment (mptap) [reply src/components/KViewSwitcher/KViewSwitcher.vue:84]: Yep, fixed.

File: src/components/KDropdownMenu/KDropdownMenu.vue (hunk 2 above)
Comment (mptap) [reply src/components/KDropdownMenu/KDropdownMenu.vue]: Great suggestion, verified after rounding up these still look ok.

**Below screenshots for DropdownMenu and Select compare Prod in Chrome on the left, with LocalDev in Firefox on the right.**

Screen Shot 2022-12-16 at 10 27 05 AM

Screen Shot 2022-12-16 at 10 26 27 AM

**Comment by mptap** [comment]
Great suggestion, verified after rounding up these still look ok.

**Below screenshots for DropdownMenu and Select compare Prod in Chrome on the left, with LocalDev in Firefox on the right.**

Screen Shot 2022-12-16 at 10 26 27 AM
Screen Shot 2022-12-16 at 10 27 05 AM

File: docs/.vitepress/components/KSkeletonBox.vue (hunk 1 above)
Comment (mptap) [reply docs/.vitepress/components/KSkeletonBox.vue]: I think they somehow got autogenerated when I ran `yarn build` and `yarn docs:preview`, but I have backed them out since they aren't needed.

**Review by adamdehaven — APPROVED** [review]
LGTM
